### PR TITLE
fix(TemplateList): show correct pagination

### LINKED ISSUE #1600: Fix DR templates pagination

### PR DESCRIPTION
**Pull request by Tommytrg**
- Fix TemplateList pagination

close #1600

### CODE REVIEW

**Comment by clbartoli** [comment]
I think the genesis_block.json file is not needed in this pr

**Review by mariocao — CHANGES_REQUESTED** [review]
I think the `genesis_block.json` shouldn't be there :smile:

File: src/components/Templates.vue [hunk 1]
```diff
@@ -79,12 +81,19 @@ export default {
   },
   computed: {
     templatesPerPage() {
-      return this.currentPage === 1 ? 8 : 9
+      return 9
```
Comment (mariocao) [inline src/components/Templates.vue:84]: Maybe a constant?

**Comment by clbartoli** [comment]
I noticed that when creating a new template the name ("New template" by default) is not deleted when writing a different name. I'm not sure if it is on purpose, if not, may be it can be fixed in this PR, if not too complicated

image

**Comment by Tommytrg** [comment]
@clbartoli When you create a new template, the editor assigns a default name: New template + N. Maybe we could discuss it on another issue if we want to change that field's current behavior. We could assign a default name but use it as a placeholder instead of as the current value.

**Review by gabaldon — APPROVED** [review]

**Pull request merged**